protocol/protocol.go

```diff
@@ -40,21 +44,56 @@ func (p *Manager) Stop() error {
 func newRouterInitializer(netAdapter *netadapter.NetAdapter, dag *blockdag.BlockDAG) netadapter.RouterInitializer {
 	return func() (*netadapter.Router, error) {
 		router := netadapter.NewRouter()
-		err := router.AddRoute([]string{wire.CmdTx}, startDummy(netAdapter, router, dag))
-		if err != nil {
-			return nil, err
-		}
+		spawn(func() {
+			err := startFlow(netAdapter, router, dag)
+			if err != nil {
+				// TODO(libp2p) Ban peer
+			}
+		})
 		return router, nil
 	}
 }
 
-func startDummy(netAdapter *netadapter.NetAdapter, router *netadapter.Router,
-	dag *blockdag.BlockDAG) chan wire.Message {
+func startFlow(netAdapter *netadapter.NetAdapter, router *netadapter.Router, dag *blockdag.BlockDAG) error {
```

Comment:
StartFlow sounds like something that starts a single flow, but if I understand correctly this starts all flows, if so - how about rename to startFlows?